staging/src/k8s.io/kubectl/pkg/cmd/auth/reconcile.go

```diff
@@ -113,15 +113,18 @@ func NewCmdReconcile(f cmdutil.Factory, streams genericclioptions.IOStreams) *co
 
 // Complete completes all the required options
 func (o *ReconcileOptions) Complete(cmd *cobra.Command, f cmdutil.Factory, args []string) error {
-	if err := o.FilenameOptions.RequireFilenameOrKustomize(); err != nil {
+	var err error
+	if err = o.FilenameOptions.RequireFilenameOrKustomize(); err != nil {
```

Comment:
revert this change, keeping scope limited is preferable when there are no assignments being made that are used later

Reply:
ok